### ISSUE TITLE
Document how to connect Kibana to an external Elasticsearch cluster

### ISSUE THREAD

**Issue by charith-elastic** [elastic/cloud-on-k8s#2001]
We currently have a standalone Kibana sample that cannot be run directly by users or tested automatically without performing an additional preparation step to setup an external Elasticsearch cluster. Instead of having this as a sample, we should just document the process of connecting to an external Elasticsearch cluster.

Related: #1875 

The standalone sample will be removed as part of #1855. For posterity, the contents of the file were:

```yaml
# This sample sets up a single Kibana instance pointing to a remote Elasticsearch cluster
apiVersion: kibana.k8s.elastic.co/v1beta1
kind: Kibana
metadata:
  name: kibana-sample
spec:
  version: 7.4.0
  config:
    elasticsearch.hosts:
    - https://url.to.elasticsearch:9200
    elasticsearch.username: my-kibana-user
    elasticsearch.password: $PASSWORD
    elasticsearch.ssl.certificateAuthorities: /mnt/certs/elasticsearch/ca.crt
  count: 1
  podTemplate:
    spec:
      containers:
      - name: kibana
        volumeMounts:
        - name: certs
          mountPath: /mnt/certs/elasticsearch
      volumes:
      - name: certs
        secret:
          secretName: my-es-http-certs
#   http:
#     service:
#       spec:
#         type: LoadBalancer # default is clusterIP
#   # inject secure settings into Kibana keystore from k8s secrets references
#   secureSettings:
#   - secretName: ref-to-secret
#   - secretName: another-ref-to-secret
#     # expose only a subset of the secret keys (optional)
#     entries:
#     - key: value1
#       path: newkey # project a key to a specific path (optional)
```

**Comment by pebrc** [comment]
We could move this into the recipes section which is not automatically executed as part of our e2e test suite.

**Comment by charith-elastic** [comment]
Fixed by #2021
